src/main/java/lotto/model/Lotto.java

```diff
@@ -0,0 +1,23 @@
+package lotto.model;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Lotto extends ValidateLottoNumbers {
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        validateLottoNumbers(numbers);
+        this.numbers = sortLottoNumbers(numbers);
+    }
+
+    private List<Integer> sortLottoNumbers(List<Integer> numbers) {
+        return numbers.stream()
+                .sorted()
+                .collect(Collectors.toList());
```

Comment:
JDK 17부터는 `toList()` 메서드를 제공합니다!
내부적으로 불변 리스트를 반환해 주기 때문에 이곳에서는 더 적절할 것 같아요!
```suggestion
                .toLIst();
```

Reply:
오 그렇군요 버전 업그레이드 기능인것을 몰랐는데 다음부터 이렇게 간단하게 표현해야겠습니다..